### ISSUE TITLE
fix(Dropdown): IE11 Dropdown tabindex lost

### ISSUE THREAD

**Issue by AlexTingkaer** [Semantic-Org/Semantic-UI-React#2327]
**Steps to Reproduce**

1. Open internet explorer 11
2. Enter https://react.semantic-ui.com/modules/dropdown#dropdown-example-selection in address line, press enter
3. Mark some or all of the headline "Selection".
4. Press the TAB button to go to the Dropdown/Select element
5. Use keyboard arrows to select any option in the list
6. Press ENTER to select an option
7. Press TAB to go to next item in the tabindex order

**Expected**
Expected to go to the **next** Dropdown or tabindex'able element

**Result**
The tab goes back to the **first** tabindex'able item on the page.

**Testcase**
 Use: https://react.semantic-ui.com/modules/dropdown#dropdown-example-selection

**Additional info**
It works as expected in Chrome 62.0.3202.94.
Maybe it is caused by the re-rendering of the component?

**Comment by layershifter** [comment]
Tested in IE11 on Windows 8. Can't reproduce, please make a reproducable example on [codesandbox.io](https://codesandbox.io)

**Comment by AlexTingkaer** [comment]
Ok, so now I have created new examples in codesandbox and codepen.

https://codesandbox.io/s/zqlpp9jk9m

https://codepen.io/anon/pen/pprXJX?editors=0011

Still with the same result - I'm not getting to the next Select, yet I am ending up with focus on the **Button**

Tested in IE 11.125.16299.0 on Windows 10.

**Comment by andrewleith** [comment]
@AlexTingkaer any luck?  I am facing the same issue and have not been able to come up with a reliable way to force the tab order to work properly.

**Comment by andrewleith** [comment]
@AlexTingkaer in case this helps you:

Out of sheer frustration I did the most counter-intuitive thing possible and somehow its working perfectly in IE11.  I am using Fomantic UI but hopefully this can help someone.  I commented out this block of code in the dropdown plugin:

```js
// prevents IE11 bug where menu receives focus even though `tabindex=-1`
if (document.activeElement.tagName.toLowerCase() !== 'input') {
  $(document.activeElement).blur();
}
```
IE11 tabbing is perfect, as is chrome (I didnt check any other browsers).
